en/lessons/basics/collections.md

```diff
@@ -3,138 +3,363 @@ version: 1.3.1
 title: Collections
 ---
 
-Lists, tuples, keyword lists, and maps.
+Tuples, Lists, Assoc Lists, Sets, Maps/Hashmaps, Seqs, and Vectors.
 
 {% include toc.html %}
 
-## Lists
+## Tuples
 
-Lists are simple collections of values which may include multiple types; lists may also include non-unique values:
+Tuples are the first kind of collection you are introduced to in Haskell. This
+is because they are simple, primitive, and have a terse, flexible, built in
+syntax. Tuples are sometimes referred to as anonymous records, but instead of
+referencing fields by name you reference them by position in the structure.
+Tuples can contain an arbitrary number of items, well this is not entirely
```

Comment:
A little awkward, but you can retain the same feel with "Although tuples can theoretically contain an arbitrary number of items, the reality is..".  If I were feeling witty I might say "the grim reality is".

Reply:
Andrew, you cheeky devil. "grim reality" it is!